DigitalLearningSolutions.Web/Controllers/LearningPortalController/SelfAssessment.cs

```diff
@@ -38,6 +38,7 @@ private IEnumerable<Competency> PopulateCompetencyLevelDescriptors(List<Competen
             return competencies;
         }
 
+        [NoCaching]

```

Comment:
Just for my benefit, why do we need NoCaching?

Reply:
I was going to link the thread but realised it was in the signposting channel you don't have access to.

This tackles the drive-through bug fix in the ticket description. In short, when going through the self assessment journey, clicking some links causes the current page content to be temporarily replaced with a loading-spinner while waiting for the response for the next page. If you reach the next page, and then hit the back button, the previous page is recovered from the browser cache to display, rather than making a call to the action. Because the cache has the latest version of the page, it has the loading spinner replace all the content on that page, making it useless until you refresh the page. NoCaching just cuts out the need to refresh, as it forces the browser to always make a call to the action.